### PR TITLE
Add General Teaching Techniques

### PR DESCRIPTION
**Pull request by ChrisOwen101**
This is a part of a document that I give to all of our new teachers with some tips about how to plan lessons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 The main goal is to inspire other teachers and share information about how to give a fun class.
 
 ## Contents:
+- General Teaching Techniques
+    - Live Coding
+    - Other Teaching Techniques
```
Comment (mkruijt) [inline README.md]: could you please add internal links to the specific headers/ sub headers?

File: README.md [hunk 2]
```diff
@@ -14,8 +17,43 @@ The main goal is to inspire other teachers and share information about how to gi
...
-    
-    
+
+
+
+
+# General Teaching Techniques
+
+## Live Coding
+
```
Comment (mkruijt) [inline README.md:27]: could you maybe add your name or the names of the teachers that help you draw up this info? In this way there is some sort of reference when people have questions about the tips that are described

**Review by mkruijt — CHANGES_REQUESTED** [review]
can you please make some small changes?

**Comment by ChrisOwen101** [comment]
@mkruijt Made those changes

**Review by mkruijt — APPROVED** [review]
cool, thanks!

**Pull request merged**